src/test/java/dk/kb/present/solr/EmbeddedSolrFieldAnalyseTest.java

```diff
@@ -33,7 +33,7 @@
 public class EmbeddedSolrFieldAnalyseTest {
 
 	private static final Logger log = LoggerFactory.getLogger(EmbeddedSolrTest.class);
-	private static String solr_home = "src/main/solr";
+	private static String solr_home = "target/solr/";
```

Comment:
here you are defining the path with a trailing slash, and below its done without it. It probably doesn't matter, but should we define them in the same way?